build.gradle

```diff
@@ -104,7 +104,7 @@ formatSource {
 }
 
 intellij {
-	plugins 'gradle', 'maven', 'terminal', 'JavaEE', 'java-i18n', 'properties'
+	plugins 'gradle', 'maven', 'terminal', 'JavaEE', 'java-i18n', 'properties', 'CSS'
```

Comment:
Does this mean that our `liferay-intellij-plugin` will require a specific version of IntelliJ?  Like the version that has the CSS Tools?  I would want to keep our plugin to have compatibility with the lower/free/community editions of Intellij.